### PR TITLE
refactor(natds-web): remove lodash dependencie from getDefaultTheme

### PR DESCRIPTION
**Pull request by yasminnvaz**
# Description

Please include a summary of the changes and features that were added. Also include which JIRA task is this related to (name and link). List any dependencies that were added or removed for this change.

## Type of change

- [ ] Feat: A new feature (non-breaking change which adds functionality)
- [x] Refactor: A code change that neither fixes a bug nor adds a feature
- [ ] Breaking change: fix or feature that would cause existing functionality to not work as expected
- [ ] Docs: Documentation only changes

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings or errors;
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] My commits follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) Guidelines;

### CODE REVIEW

**Review by lauralucca — APPROVED** [review]
LGTM

**Comment by arielwb** [comment]
eu achei estranho que a versão subiu pra 2, acho que é devido ao tipo do commit estar refactor? não sei, mas esse PR deveria fazer um bump do tipo PATCH.

File: lerna.json [hunk 1]
```diff
@@ -36,5 +36,5 @@
     "packages/react"
   ],
   "useWorkspaces": true,
-  "version": "1.8.3"
+  "version": "2.0.0-alpha.DSY-999.1.0"
```
Comment (arielwb) [inline lerna.json:39]: eu achei estranho que a versão subiu pra 2, acho que é devido ao tipo do commit estar refactor? não sei, mas esse PR deveria fazer um bump do tipo PATCH.

**Review by arielwb — APPROVED** [review]

File: lerna.json (hunk 1 above)
Comment (yasminnvaz) [reply lerna.json:39]: estranho mesmo, não tinha reparado. Mas mesmo como refactor, não deveria subir a major, né? 🤔  vou investigar

**Pull request merged**